aws-auth-cognito/src/test/java/featureTest/utilities/AuthOptionsFactory.kt

```diff
@@ -74,6 +76,17 @@ object AuthOptionsFactory {
         AuthAPI.getVersion -> TODO()
     } as T
 
+    private fun getSignInOptions(optionsData: JsonObject): AuthSignInOptions {
+        return if (optionsData.containsKey("signInOptions")) {
+            val authFlowType = AuthFlowType.valueOf(
+                ((optionsData["signInOptions"] as Map<String, String>)["authFlow"] as JsonPrimitive).content
+            )
+            AWSCognitoAuthSignInOptions.builder().authFlowType(authFlowType).build()
```

Comment:
Is metadata not required for custom auth?

Reply:
metadata is not required for custom auth yes.